pgmpy/readwrite/ProbModelXML.py

```diff
@@ -518,6 +551,27 @@ def add_probnet_additionalconstraints(self, constraint):
             argument_value = argument.attrib['value']
             self.probnet['AdditionalConstraints'][constraint_name][argument_name] = argument_value
 
+    def add_criterion(self, criterion):
+        """
+        Adds Decision Criteria to the probnet dict.
+
+        Parameters
+        ----------
+        criterion: etree Element consisting DecisionCritera tag.
+
+        Examples
+        -------
+        >>> reader = ProbModelXMLReader()
+        >>> reader.add_criterion(criterion)
```

Comment:
Please add over here what is ```criterion``` variable. Documentation means that user is able to understand what does this piece of code means, write some examples.